.docsettings.yml

```diff
@@ -2,9 +2,8 @@ omitted_paths:
   - tools/*
   - scripts/*
   - azure-mgmt-netapp/tests/*
-  - "azure-mgmt*/*"
```

Comment:
I assume these entries were intentionally removed?

Reply:
yes, there is no packages at the root of the repo, that's legacy from when we were transitioning into "sdk" folder